src/Compilers/VisualBasic/Portable/Symbols/Source/SynthesizedFieldSymbol.vb

```diff
@@ -59,6 +59,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 Return
             End If
 
+            Dim compilation = Me.DeclaringCompilation
+            If Type.ContainsTupleNames() AndAlso
+                compilation.HasTupleNamesAttributes AndAlso
```

Comment:
Codeflow didn't thread properly, so manually moving my comment overto this thread:

IIRC, I think that diagnostic is reported elsewhere. This check prevents errors, should we ever get this far (which I believe is possible in the APIs).